olive/evaluator/olive_evaluator.py

```diff
@@ -337,9 +338,11 @@ def _inference(
             # keep as numpy or torch arrays
             preds.append(outputs.cpu())
             targets.append(labels.cpu())
+            logits.append(result.cpu())
```

Comment:
result is not a torch tensor if the `len(output_names) >1`. It is going to be a dictionary of tensors.

Reply:
seconds this comment

Reply:
Updated.